build.sbt

```diff
@@ -36,17 +35,4 @@ pgpPassphrase := Option(System.getenv("SONATYPE_GPG_PASSPHRASE"))
 
 publicMvnPublish
 
-unmanagedSourceDirectories.in(Compile) += (Dependencies.playWsJson(scalaVersion.value).head.revision match {
-  case Dependencies.playJson24 =>
-    sourceDirectory.value / "main" / "play_json_2.4"
-  case Dependencies.playJson27 =>
-    sourceDirectory.value / "main" / "play_json_2.7"
-  case _ =>
-    throw new Exception("Unsupported Play JSON version")
-})
-
-name := (Dependencies.playWsJson(scalaVersion.value).head.revision match {
-  case Dependencies.playJson24 => s"${name.value}_playjson24"
-  case Dependencies.playJson27 => s"${name.value}_playjson27"
-  case _ => throw new Exception("Unsupported Play JSON version")
-})
+name := s"${name.value}_playjson27"
```

Comment:
I would like this `27` be generated from the play json version we actually use in the code.
I already see play json being bumped and production going down.

Reply:
Something like:
```suggestion
name := s"${name.value}_playjson${Dependencies.playVersion.split('.').take(2).mkString}"
```